### ISSUE TITLE
NH: Posters of (new) villagers should not require amiibo

### ISSUE THREAD

**Issue by sharris40** [jefflomacy/villagerdb#249]
All posters have the same, generic line for the sources section that says amiibo must be scanned to obtain the poster. This is inaccurate. amiibo are not required for any normal residents, since you can add them from the menu if they live in your town. Villagers new to New Horizons don’t even have amiibo to scan, so they can only be added from the menu. Only characters that cannot become normal residents (and the Smash Bros. Villager) need amiibo to obtain the posters.

**Comment by jefflomacy** [comment]
True. I'll get this updated in the spreadsheet. We may just adjust all the items to say Harv's Island since it is pretty common knowledge how these are obtained.
